packages/eslint-plugin/src/configs/README.md

```diff
@@ -0,0 +1,42 @@
+# Premade configs
+
+These configs exist for your convenience. They contain configuration intended to save you time and effort when configuring your project by disabling rules known to conflict with this repository, or cause issues in typesript codebases.
+
+## All
+
+TODO when all config is added.
+
+## Recommended
+
+The recommended set is an **_opinionated_** set of rules that we think you should use because:
+
+1. They help you adhere to Typescript best practices.
```

Comment:
Cringing at all the `Typescript` casing (instead of **TypeScript**) 😂

Reply:
It's such a pain pressing shift halfway through a word.... haha